pyscript.core/src/stdlib/pyscript/websocket.py

```diff
@@ -36,9 +37,11 @@ def __init__(self, **kw):
             socket = js.WebSocket.new(url)
         object.__setattr__(self, "_ws", socket)
 
+        set_attr = js.Reflect.set
         for t in ["onclose", "onerror", "onmessage", "onopen"]:
             if t in kw:
-                socket[t] = kw[t]
+                # Pyodide fails at setting socket[t] directly
+                set_attr(socket, t, create_proxy(kw[t]))
```

Comment:
Python has a builtin called `setattr` that does this:
```suggestion
                setattr(socket, t, create_proxy(kw[t]))
```

Reply:
silly me ... thanks, fixed that! why wouldn't that accessor be set though? MicroPython had no issue with `socket[t] = kw[t]` but Pyodide threw an error before due "unable to set property X" issue 🤔

we are good with `setattr` but it's a bit of a worry/need to remember thing I haven't witnessed before.